libs/payments/cart/src/lib/cart.utils.spec.ts

```diff
@@ -22,4 +33,74 @@ describe('utils', () => {
       expect(result).toBe(CartErrorReasonId.Unknown);
     });
   });
+
+  describe('cartEligibilityDetailsMap', () => {
```

Comment:
No need for a spec to test static values. All this does is duplicate the static values here, effectively.

Reply:
I'll remove. Thank you!